src/bridge.ts

```diff
@@ -109,18 +104,18 @@ export const initializeRoutes = (app: express.Server, serviceUrl: string, botUrl
         }
         else {
             // Conversation was never initialized
-            res.status(400).send;
+            res.status(400).send();
         }
     })
 
-    app.post('/v3/directline/conversations/:conversationId/upload', (req, res) => { console.warn("/v3/directline/conversations/:conversationId/upload not implemented") })
-    app.get('/v3/directline/conversations/:conversationId/stream', (req, res) => { console.warn("/v3/directline/conversations/:conversationId/stream not implemented") })
+    router.post('/v3/directline/conversations/:conversationId/upload', (req, res) => { console.warn("/v3/directline/conversations/:conversationId/upload not implemented") })
```

Comment:
I think all these `console.warns` could be simplified by using `req.url` instead of duplicated endpoint. This allows them to be correct regardless of where the Router is attached.

Reply:
that should clean things up a bit! 

I intentionally left the stubbed endpoints to indicate the functionality of directline that this package does not implement